### DISCUSSION TITLE
Getting "No Network" no matter what I've tried.

### DISCUSSION THREAD

**Discussion by Smokex365** [tteck/Proxmox#3871] [Help Needed]
I've been trying to use the helper scripts, specifically the unifi script to start with  (have tried others as well like tautulli) and I can not get past the No Network error no matter what.

- ipv6 is disabled on my router (pfsense)
- manual `hostname -I `returns the node ip
- 1.1.1.1 is pingable as well as github.com
- nslookup also resolves one of the correct ip's
- the container(s) are pulling ip's from dhcp *and* registering hostnames and dns entries on my router
- going through advanced settings and logging in the failed containers with all the same checks working

I'm at a loss. Both the host and the failed containers have connectivity but the script still errors out every time whether through the default or advanced on multiple nodes.

**Comment by Mellowlynx** [comment]
Probably DNS, can you ping something like google.com or other domain?

**Reply by Smokex365** [reply]
Yes, I can ping everything by ip and domain.

**Comment by tteck** [comment]
It is a simple snippet

https://github.com/tteck/Proxmox/blob/b7fe85cd5b1dfc0dab089a7cb6f40e8aa364f7c7/misc/install.func#L10-L11

https://github.com/tteck/Proxmox/blob/b7fe85cd5b1dfc0dab089a7cb6f40e8aa364f7c7/misc/install.func#L96-L107

It allows 30 seconds for the IP.

**Comment by thelazurus** [comment]
I'm in the same boat, I'm able to ping everything and I know the server has good internet. I know last time I ran scripts I was able to tell it to proceed anyway but it seems that's not happening anymore.

**Reply by tteck** [reply]
Are you using SSH?

**Reply by thelazurus** [reply]
I was just using the webui shell. However, I managed to get things to work for me by switching from the debian docker script to the alpine one. I tried manually setting up a Debian LXC to test something else and never got anything from it, I wonder if something is wrong with the template

**Reply by tteck** [reply]
Remove the template and redownload it.
